samples/word/99-preview-apis/manage-change-tracking.yaml

```diff
@@ -0,0 +1,155 @@
+id: word-manage-change-tracking
+name: Track changes
+description: This sample shows how to get and set the ChangeTrackingMode and get reviewed text.
+host: WORD
+api_set:
+    WordApi: '1.4'
+script:
+    content: |
+        $("#get-change-tracking-mode").click(() => tryCatch(getChangeTrackingMode));
+        $("#set-change-tracking-mode").click(() => tryCatch(setChangeTrackingMode));
+        $("#get-reviewed-text").click(() => tryCatch(getReviewedText));
+        $("#setup").click(() => tryCatch(setup));
+
+        /** Get the current ChangeTrackingMode. */
```

Comment:
Similar to an earlier comment, I think that using "track changes" or "tracking mode" (UI terms) helps make the comments more readable. The enum name is in the code, so people searching that way can find it.